spec/lib/locasms/client_spec.rb

```diff
@@ -57,25 +57,25 @@
     end
 
     it 'Should not send SMS' do
-      subject.should_receive(:numbers)
+      expect(subject).to receive(:numbers)
         .once
         .with([:a, :b, :c])
         .and_raise(LocaSMS::Exception)
 
-      LocaSMS::Helpers::DateTimeHelper.should_receive(:split)
+      expect(LocaSMS::Helpers::DateTimeHelper).to receive(:split)
         .once
         .with(:datetime)
         .and_return(%w[date time])
 
-      rest_client.should_receive(:get).never
+      expect(rest_client).to receive(:get).never
 
-      lambda{ subject.deliver_at('given message', :datetime, :a, :b, :c) }.should raise_error(LocaSMS::Exception)
+      expect{ subject.deliver_at('given message', :datetime, :a, :b, :c) }.to raise_error(LocaSMS::Exception)
```

Comment:
Line is too long. [109/80]<br>Space missing to the left of {.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.